typescript/sdk/src/providers/transactions/schemas.ts

```diff
@@ -1,7 +1,17 @@
 import { z } from 'zod';
 
+import { ZHash } from '../../metadata/customZodTypes.js';
+
 export const PopulatedTransactionSchema = z.object({
   to: z.string(),
   data: z.string(),
   chainId: z.number(),
 });
+
+export const BigNumberSchema = z.any();
```

Comment:
I know this has been discussed before but can you clarify why we would want to allow non-primitive, lib-specific BigNumbers in our configs?

Reply:
afaik we do not want this.. ideally we would have this: https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/3771

line 11 exists here just to unblock me with getting these zod schemas completed

Reply:
the workaround should be z.string() for any "bignumberish" types before 3771